ng-ncc-app/src/app/pages/auth/auth.component.html

```diff
@@ -0,0 +1,3 @@
+<p>
```

Comment:
Is this supposed to be here?

Reply:
The file itself is necessary because of it being a component, but the template itself is never used.